G/GlibNetworking/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GlibNetworking"
+version = v"2.74.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/glib-networking/2.74/glib-networking-$(version).tar.xz", "1f185aaef094123f8e25d8fa55661b3fd71020163a0174adb35a37685cda613b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/glib-networking-*
+install_license COPYING
+
+mkdir build-glib && cd build-glib
+meson --cross-file=${MESON_TARGET_TOOLCHAIN} --buildtype=release ..
+
+# Meson beautifully forces thin archives, without checking whether the dynamic linker
+# actually supports them: <https://github.com/mesonbuild/meson/issues/10823>.  Let's remove
+# the (deprecated...) `T` option to `ar`
+sed -i.bak 's/csrDT/csrD/' build.ninja
+
+ninja -j${nproc}
+
+# Darwin products are also .so instead of .dylib so we need to rename
+mkdir -p ${libdir}/gio/modules
+cp proxy/environment/libgioenvironmentproxy.so ${libdir}/gio/modules/libgioenvironmentproxy.${dlext}
+cp tls/gnutls/libgiognutls.so ${libdir}/gio/modules/libgiognutls.${dlext}
```

Comment:
Are you sure this is correct?  These are usually plugins which ***need*** to be called `*.so`.

Reply:
No. I could not make OSX work. As I said above, neither `.so` nor `.dylib` worked. Maybe I should just exclude OSX for now.

Reply:
Can you elaborate on what "work" means here?

Reply:
Soup could not build on OSX even if `libgiognutls` and `libgioenvironmentproxy` were both in `lib/gio/modules` (and both `.so` and `.dylib` were there). The error was the same as when there was no `glib-networking`.

Reply:
https://docs.binarybuilder.org/stable/troubleshooting/#General-comments
> The best recommendation is to access the build environment and carefully read the log files generated by the build systems: it is not uncommon that build systems would only print to screen misleading error messages, and the actual problem is completely different (e.g. "library XYZ can't be found", when the problem instead is that the command they run to look for library XYZ fails for unrelated reasons, for example for a wrong compiler flag used in the check). Having an understanding of what the build system is doing would also be extremely useful.